dayton/spiders/reap_spider.py

```diff
@@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
     def getPaymentPlan(self, response):
         sel = Selector(response)
         item = response.meta['item']
-        item['paymentplan'] = False
-        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
-            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
+        item['payment_plan'] = False
+        
+        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
+            try:
```

Comment:
Ah, yeah. I was testing out some things locally, I had so many things that
I didn't commit, I missed that line. I'd never staged changes before.

The try block should be removed, since the exception I used to be catching
should happen anymore.

On Thu, May 14, 2015 at 9:01 PM, David Best <notifications@github.com>
wrote:

> In dayton/spiders/reap_spider.py
> <https://github.com/codefordayton/scrapers/pull/7#discussion_r30378346>:
>
> > @@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
> >      def getPaymentPlan(self, response):
> >          sel = Selector(response)
> >          item = response.meta['item']
> > -        item['paymentplan'] = False
> > -        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
> > -            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
> > +        item['payment_plan'] = False
> > +
> > +        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
> > +            try:
>
> I get an error about an unexpected unindent. I can fix it by removing the
> try: line. Should this have an except block?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codefordayton/scrapers/pull/7/files#r30378346>.
>